src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -32,29 +32,42 @@
 /**
  * Edits the details of an existing person in the address book.
  */
-public class EditCommand extends Command {
+public class EditCommand extends UndoableCommand {
 
     public static final String COMMAND_WORD = "edit";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the details of the person identified "
-            + "by the name or NRIC of the patient.\n"
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the details of the Person identified "
+            + "by the name or NRIC of the Patient.\n"
             + "Existing values will be overwritten by the input values.\n"
             + "Format: edit n/NAME or id/IC_NUMBER [Fields] ...\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
             + PREFIX_PHONE + "91234567";
 
-    public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
-    public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided. Example: p/98742122.";
-    public static final String MESSAGE_PERSON_NOT_FOUND =
-            "The given combination of Name and NRIC does not match any person in the patients list.";
+    public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Patient: %1$s";
+    public static final String MESSAGE_UNDO_PERSON_SUCCESS = "Undoing the Editing of Patient:  %1$s";
```

Comment:
Just nitpicking here a little, but perhaps the variable could be named `MESSAGE_UNDO_EDIT_PERSON_SUCCESS` to be alined with the variable name in `DeleteCommand` and `AddCommand`

Reply:
Again, sorry i highlighted 2 lines, I am referring to line 48